lib/susedistribution.pm

```diff
@@ -168,5 +166,55 @@ sub become_root() {
     send_key('ctrl-l');
 }
 
+# initialize the consoles needed during our tests
+sub activate_consoles {
+    my ($self) = @_;
+    if (get_var('BACKEND', 'qemu')) {
```

Comment:
not sure what you mean. And no, the code/data can't move to os-autoinst as it's distribution specific